tests/ttnn/unit_tests/gtests/tensor/test_sharding_with_alignment.cpp

```diff
@@ -844,15 +844,15 @@ INSTANTIATE_TEST_SUITE_P(
                     }
             },
             CreateShardedTensorWithAlignmentExpected{
-                .physical_size = Size{20, 20}
+                .physical_size = Size{20, 5}
             }
         },
         // Example 2b: Logical shard shape that is already aligned
         // NOTE: ShardMode::PHYSICAL is equivalent in this case
-        // - Along width: 5 / 4 is 2 shards; 2 * 4 = 8
+        // - Along width: 8 / 4 is 2 shards; 2 * 4 = 8
         CreateShardedTensorWithAlignmentParams{
             CreateShardedTensorWithAlignmentInputs{
-                .shape = SimpleShape{1, 2, 10, 5},
+                .shape = SimpleShape{1, 2, 10, 8},
                 .data_type = DataType::UINT8,
                 .page_config = PageConfig(Layout::ROW_MAJOR),
```

Comment:
Why change 5 to 8? We support partial shards. Can you keep old specs for this test?